app-backend/api/src/main/scala/user/Auth0User.scala

```diff
@@ -323,6 +323,34 @@ object Auth0Service extends Config with LazyLogging {
       .flatMap { responseAsAuth0User _ }
   }
 
+  def createERUser(
```

Comment:
Similar to above, might be helpful to make this `createAnonymizedUser` instead, so that the name reflects what it's doing